conf/httpd.conf-example

```diff
@@ -36,7 +36,7 @@ WSGIDaemonProcess example.com \
     processes=5 \
     threads=1 \
     display-name=example.com \
-    python-path=/data/vhost/example.com/repository:/data/vhost/example.com/virtualenv-repository/lib/python2.7/site-packages
+    python-path=/data/vhost/example.com/repository:/data/vhost/example.com/virtualenv-repository/lib/python3/site-packages
```

Comment:
Set `python-home` to the root of the virtualenv, then `python-path` to the root of the Django project, so something like:
```
WSGIDaemonProcess example.com \
  ...
  python-home=/data/vhost/example.com/venv \
  python-path=/data/vhost/example.com/repository

```

Reply:
You can see examples of this in the internal servers repo under `vhosts/` - check out Global MapIt again, or CAPE.

Reply:
hmmm, might need some help with this one. 

global mapit for instance is: 

```
WSGIDaemonProcess !!(*= $vhost *)!! \
  user=!!(*= $user *)!! \
  group=!!(*= $group *)!! \
  processes=5 \
  threads=1 \
  display-name=!!(*= $vhost *)!! \
  python-home=/data/vhost/!!(*= $vhost *)!!/!!(*= $vcspath *)!!/.venv/ \
  python-path=/data/vhost/!!(*= $vhost *)!!/!!(*= $vcspath *)!!
```

Can this just be copied in directly or is this the old one?

Reply:
As it's just an example, the exact paths used should just conform to the rest of the example paths. `python-home` should just point to the root of the venv (not the `site-packages` subdirectory, as everything is under this directory - [see notes here](https://modwsgi.readthedocs.io/en/master/configuration-directives/WSGIPythonHome.html)) and `python-path` just needs to point to the Django project root, so that the modules there can be found ([see notes here](https://modwsgi.readthedocs.io/en/master/configuration-directives/WSGIPythonPath.html)). We'll provide the deploy-specific config in the internal servers repo. Does this make sense?

Reply:
OK, I'm going to say that's done. Should be ready for you.